MediaBrowser.WebDashboard/dashboard-ui/autoorganizelog.html

```diff
@@ -18,8 +18,8 @@
                     <div class="listTopPaging" style="float: left; position: relative; top: 15px;">
                     </div>
 
-                    <!--<button style="display: none;" class="btnClearLog" type="button" data-inline="true" data-icon="forbidden" data-mini="true">Clear</button>-->
                     <div style="float: right; position: relative; top: 15px;margin-top: -5px;display:none;" class="organizeTaskPanel">
+                        <button type="button" class="btnClearLog" data-icon="forbidden" data-mini="true" data-inline="true" style="display: none;">Clear</button>
```

Comment:
This isn't localizable. Should probably be a new string added to the files in https://github.com/Nodja/MediaBrowser/tree/patch-2/MediaBrowser.Server.Implementations/Localization/Server